Assets/GothicVR/Scripts/Manager/Culling/VobMeshCullingManager.cs

```diff
@@ -15,11 +15,14 @@ namespace GVR.Manager.Culling
     /// </summary>
     public class VobMeshCullingManager : SingletonBehaviour<VobMeshCullingManager>
     {
+        [SerializeField]
+        protected bool drawGizmos;
```

Comment:
We put this kind of setting into FeatureFlags.
There should be a section of _Developer_ flags.

Please put it there and name it like _debugDrawCullingGizmosOnVobs_ to have it centrally available.

Reply:
Last point on the PR. Then I merge. I can also do the change for you. Just tell me what's faster. @ThomasMountainborn

Reply:
### Can you do it? Thanks!